src/pop/crypto/progpow/progpow.cpp

```diff
@@ -2,6 +2,11 @@
 // https://www.veriblock.org
 // Distributed under the MIT software license, see the accompanying
 // file LICENSE or http://www.opensource.org/licenses/mit-license.php.
+#include <cstdint>
+#include <mutex>
+#include <utility>
+#include <vector>
+#include <veriblock/pop/algorithm.hpp>
```

Comment:
use #include "file.h" for project includes

Reply:
fixed